cmd/flags_misc.go

```diff
@@ -145,7 +144,12 @@ func ParseMiscFlags(ctx context.Context, cmd *cobra.Command) (context.Context, e
 			mux.HandleFunc("/debug/pprof/profile", pprof.Profile)
 			mux.HandleFunc("/debug/pprof/symbol", pprof.Symbol)
 			mux.HandleFunc("/debug/pprof/trace", pprof.Trace)
-			log.Println(http.ListenAndServe("0.0.0.0:6000", mux))
+			err := http.ListenAndServe("0.0.0.0:6000", mux) //nolint:gosec
+			if err != nil {
+				log.Errorw("failed to start pprof server", "err", err)
+			} else {
+				log.Info("started pprof server on port 6000")
+			}
```

Comment:
Let's panic here. No need to log it

Reply:
Agreed that Errorw is the wrong approach, but should we maybe do a log.Fatal instead? Or just returning the error which will bubble up anyways?

Do you share the same opinion as here, that panics are not relevant for users because of the stack trace?
https://stackoverflow.com/questions/35996966/what-are-the-differences-in-outcome-for-panic-vs-log-fatalln

Reply:
Commiting as "Fatalw" for now but will adapt based on this discussion

Reply:
Fatal works